src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -0,0 +1,3394 @@
+          onAttack {
+            damage 70
+          }
+        }
+      };
+      case GALARIAN_FARFETCH_D_87:
+      return basic (this, hp:HP010, type:C, retreatCost:2) {
+        weakness Y
+        resistance F, MINUS30
+        bwAbility "TBA", {
+          text "TBA"
+          actionA {
+          }
+        }
+        move "TBA", {
+          text "100 damage. TBA"
+          energyCost C, C, C
+          attackRequirement {}
+          onAttack {
+            damage 100
+          }
+        }
+      };
+      case GALARIAN_SIRFETCH_D_88:
+      return basic (this, hp:HP010, type:C, retreatCost:2) {
+        weakness Y
+        resistance F, MINUS30
+        bwAbility "TBA", {
+          text "TBA"
+          actionA {
+          }
+        }
+        move "TBA", {
+          text "100 damage. TBA"
+          energyCost C, C, C
+          attackRequirement {}
+          onAttack {
+            damage 100
+          }
+        }
+      };
+      case LARVITAR_89:
+      return basic (this, hp:HP070, type:F, retreatCost:1) {
+        weakness G
+        move "Sand Spray", {
+          text "10 damage. "
+          energyCost F
+          attackRequirement {}
+          onAttack {
+            damage 10
+          }
+        }
+        move "Pierce", {
+          text "20 damage. "
+          energyCost C, C
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+      };
+      case PUPITAR_90:
+      return evolution (this, from:"Larvitar", hp:HP080, type:F, retreatCost:3) {
+        weakness G
+        move "Sand Spray", {
+          text "20 damage. "
+          energyCost F
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+        move "Rocket Evolution", {
+          text "40 damage. Search your deck for a card that evolves from this Pokémon and play it on top of this Pokémon to evolve it. Then, shuffle your deck."
+          energyCost C, C, C
+          attackRequirement {}
+          onAttack {
+            damage 40
+          }
+        }
+      };
+      case TYRANITAR_91:
+      return evolution (this, from:"Pupitar", hp:HP180, type:F, retreatCost:4) {
+        weakness G
+        move "Bedrock Breaker", {
+          text "120 damage. Discard a Stadium in play."
+          energyCost F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 120
+          }
+        }
+        move "Mountain Swing", {
+          text "250 damage. Discard 5 cards from the top of your deck."
+          energyCost F, F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 250
+          }
+        }
+      };
+      case TRAPINCH_92:
+      return basic (this, hp:HP060, type:F, retreatCost:1) {
+        weakness G
+        move "Sand Impulse", {
+          text "20 damage. This attack does 10 damage to each of your opponent’s Benched Pokémon. (Don’t apply Weakness and Resistance.)"
```

Comment:
Looks like this is 10+ damage

[Trapinch](https://www.pokebeach.com/news/0520/038182_P_NAKKURA.jpg)